TelepresenceBot/mobile/build.gradle

```diff
@@ -36,5 +36,8 @@ dependencies {
     compile 'com.android.support:recyclerview-v7:25.0.1'
     compile 'com.android.support:support-annotations:25.0.1'
     compile 'com.github.felHR85:UsbSerial:4.3'
+    compile('io.socket:socket.io-client:0.8.2') {
```

Comment:
OkHttp also has websockets module. I wonder what is the difference. 
I wondered about it also because of the json dependency of this socket.io below.